ext/pdo/pdo_stmt.stub.php

```diff
@@ -29,25 +29,25 @@ public function errorCode() {}
     public function errorInfo() {}
 
     /** @return bool */
-    public function execute(?array $input_parameters = null) {}
+    public function execute(?array $params = null) {}
 
     /** @return mixed */
-    public function fetch(int $mode = PDO::FETCH_BOTH, int $cursor_orientation = PDO::FETCH_ORI_NEXT, int $cursor_offset = 0) {}
+    public function fetch(int $mode = PDO::FETCH_BOTH, int $cursorOrientation = PDO::FETCH_ORI_NEXT, int $cursorOffset = 0) {}
 
     /** @return array */
     public function fetchAll(int $mode = PDO::FETCH_BOTH, mixed ...$args) {}
 
     /** @return mixed */
-    public function fetchColumn(int $index = 0) {}
+    public function fetchColumn(int $column = 0) {}
 
     /** @return mixed */
-    public function fetchObject(?string $class = "stdClass", ?array $constructor_args = null) {}
+    public function fetchObject(?string $class = "stdClass", ?array $ctorArgs = null) {}
```

Comment:
Is there a reason why this can't be `$constructorArgs`?

Reply:
It could also be a $constructorArgs. I prefer the shorter version a bit, but don't insist on it :)

Reply:
IMHO `args` is a quite common abbreviation but `ctor` is much less common. I'd prefer `$constructorArgs`